include/seqan3/alignment/pairwise/detail/pairwise_alignment_algorithm_banded.hpp

```diff
@@ -253,7 +261,7 @@ class pairwise_alignment_algorithm_banded :
 
         this->track_last_column_cell(*alignment_column_it, *cell_index_column_it);
 
-        for (; first_row_index < sequence2_size; ++first_row_index)
+        for (; first_row_index < std::min(sequence2_size, row_size); ++first_row_index)
```

Comment:
```suggestion
        for (size_t last_row = std::min(sequence2_size, row_size); first_row_index < last_row; ++first_row_index)
```

This min is "technically" called in every loop iteration